### ISSUE TITLE
front matter title not used for html export 

### ISSUE THREAD

**Issue by mahnu** [shd101wyy/markdown-preview-enhanced#631]
Thanks again for your great work!

It seems that the front matter "title" tag is not used for html export.
I am using mde version 0.14.10 (on debian stretch).

E.g. if testtitle.md document starts with:
```
---
title: "This is my title"
--- 
```

The html document (offline for instance) use the file name as html title. Here is what you can find the html output file:
```
<!DOCTYPE html>
  <html>
    <head>
      <title>testtitle</title>
...
```

I would expect:
```
<!DOCTYPE html>
  <html>
    <head>
      <title>This is my title</title>
...
```

It seems that the same bug occurs for phantomjs export to PDF.
Did I miss something or can you confirm this bug?

**Comment by shd101wyy** [comment]
Sure, will do.

**Comment by shd101wyy** [comment]
@mahnu Done. Cheers :)

**Comment by mahnu** [comment]
Thanks again!  Great work.
I tested and confirm this issue is solved in 0.14.11 → closing this issue.
